server/index.js

```diff
@@ -36,5 +36,5 @@ module.exports = function Server(config, onStart) {
   };
 
   // Construct and return mockyeah API
-  return Object.assign({ server }, app.routeManager, { use, config, close });
+  return Object.assign({ server }, app, app.routeManager, { use, config, close });
```

Comment:
Wasn't sure if we prefer to merge this directly (risk of namespace conflicts), or make it `mockyeah.app`.

Reply:
I believe app instance conflicts with mock route methods (i.e. `.[get,post,put,delete,all]()`).